src/main/common/huffman_table.c

```diff
@@ -0,0 +1,285 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdint.h>
+
+#include "huffman.h"
+
+/*
+ * Huffman Table, used to compress a bytestream
+ */
+const huffmanTable_t huffmanTable[HUFFMAN_TABLE_SIZE] = {
```

Comment:
This table can be optimized a bit by setting first free bit and using bitcounting functions to get code length
```
marker = code & (0-code);
len = __builtin_clz (marker);
stripped_code = code ^ marker;  // clear marker bit
```
You can use macro to hide implementation:
```
#define ROW(len,code) ((code) | (0x8000 >> (len))